packages/automerge-repo/src/DocHandle.ts

```diff
@@ -490,9 +490,9 @@ export interface DocHandleEvents<T> {
   change: (payload: DocHandleChangePayload<T>) => void
   delete: (payload: DocHandleDeletePayload<T>) => void
   unavailable: (payload: DocHandleDeletePayload<T>) => void
-  "ephemeral-message": (payload: DocHandleEphemeralMessagePayload) => void
+  "ephemeral-message": (payload: DocHandleEphemeralMessagePayload<T>) => void
```

Comment:
Ah, I think this is wrong. The payload is not <T> (the doc type) but a different message type.

Reply:
Oh, no, I see, nevermind.